tests/integration/explain/debug/fixture.go

```diff
@@ -14,9 +14,13 @@ type dataMap = map[string]any
 
 var basicPattern = dataMap{
 	"explain": dataMap{
-		"selectTopNode": dataMap{
-			"selectNode": dataMap{
-				"scanNode": dataMap{},
+		"operationNode": []dataMap{
```

Comment:
question: Just to confirm even the bare bones / basic most query will use an `operationNode` from now on?

Reply:
Yes that's correct